deploy/crds/smartgateway.infra.watch_smartgateways_crd.yaml

```diff
@@ -94,9 +94,11 @@ spec:
                 To avoid the round trip for every message, the use of prefetch can be used to allow the receiver to request messages be sent
                 in anticipation of them being sent to us.
               type: integer
-            containerImagePath:
-              description: Path to the container image this Operator will deploy. Value is defined as the standard registry/image_name:tag
-                format.
+            metricsContainerImagePath:
```

Comment:
I see these new configuration options, but I don't see where they are being used?

Reply:
https://github.com/infrawatch/smart-gateway-operator/blob/46922f924f1301f8b7303b2e80fcf9ad5b8cbf92/roles/smartgateway/templates/deployment.yaml.j2#L45

and

https://github.com/infrawatch/smart-gateway-operator/blob/46922f924f1301f8b7303b2e80fcf9ad5b8cbf92/roles/smartgateway/templates/deployment.yaml.j2#L57

Reply:
However, since pretty soon both functionalities will be in the same container image, we can adjust this to fit with the current CRD that only specifies one containerImagePath. On the otherhand, separating them out may be convenient for testing purposes